### DISCUSSION TITLE
tailwindcss CDN broken

### DISCUSSION THREAD

**Discussion by MarinosTBH** [tailwindlabs/tailwindcss#13363] [Help]
im using tailwindcss cdn and it looks like a bad gateway error, 
does others have the same prob ? 

https://cdn.tailwindcss.com/

**Comment by alex-krasikau** [comment]
Hey @MarinosTBH!

It works well for me. Please try it again and ensure you turn off your VPN if you are using one.

**Comment by MarinosTBH** [comment]
It works now!
Been off all day long for me and my team.
And i don't use vpn!
